mint.json

```diff
@@ -104,7 +104,7 @@
       }
     }
   },
-  "openapi": "https://api.sync.so/api-json",
+  "openapi": "openapi.json",
```

Comment:
why not using dynamically from prod deploy? how is it generated now?

Reply:
Dynamically from prod makes sense ideally, but right now since docs&playground for endpoints are generated directly from open api spec, I've created a static file for now to first improve docs.

I'm assuming the open-api.json is being generated in API service right now. We need to update that to match this and we can then switch to referring to url.

Reply:
The current static openapi.json is copy of the prod spec with modifications made to it.